### PR TITLE
DL: Inline text

### PR DESCRIPTION
**Pull request by wesoudshoorn**
This PR adds inline-text styling for default elements, and adds utility classes.

It is based on the #34 because I needed the @extend utility classes.

### CODE REVIEW

**Comment by jvanbaarsen** [comment]
@wesoudshoorn What does the DL prefix stand for in the PR title?

**Comment by jvanbaarsen** [comment]
This cannot be merged in the way it is now, since you have based your change of another PR which is currently still being discussed. Can you drop that commit from your PR?

File: src/elements/typography.css [hunk 1]
```diff
@@ -20,3 +20,33 @@ ul {
 a {
   color: var(--color-grey-darkest);
 }
+
+b,
+strong {
```
Comment (jvanbaarsen) [inline src/elements/typography.css]: I actually don't think we need to add this. Browsers make text bold by default.

**Comment by wesoudshoorn** [comment]
The DL stands for design-language, CL for component-library.

**Comment by jvanbaarsen** [comment]
@wesoudshoorn Ok, I would be fine with not adding them (Since you already create the PRs in their own repo's).

**Comment by jvanbaarsen** [comment]
@wesoudshoorn I'm going to do a pass-over to convert the utilities more in line with the "Utility CSS" direction we are aiming for. I'll do that in a separate commit so we can revert it if it becomes contested.

**Comment by jvanbaarsen** [comment]
@wesoudshoorn I split up a lot of the utilities. Can you let me know your thoughts on this? I want to merge this around after lunch.

**Comment by wesoudshoorn** [comment]
Looks good @jvanbaarsen

**Pull request merged**